core/domain/exp_services.py

```diff
@@ -1506,7 +1567,12 @@ def save_new_exploration(
                 'category': exploration.category,
             })])
     user_services.add_created_exploration_id(committer_id, exploration.id)
-    user_services.add_edited_exploration_id(committer_id, exploration.id)
+    user_contribution_models = user_services.add_edited_exploration_id(
```

Comment:
>> Again, this should follow the pattern (and be consistent for both created + edited exp ids):
>>
>>    domain_object = get_from_model()
>>    domain_object.add_edited_exploration_id()
>>    save_domain_object_to_datastore()
>>
> Done. This actually returns a domain object. I've updated the name per suggestion.

If this returns a domain object, why is the result called user_contribution_models?

Also why is there a difference between how the above line and this line are treated (it seems the return value of this line matters vs the one above doesn't)?

I think that the earlier comment still applies, i.e. the logic should still be

```
    domain_object = get_from_model()
    domain_object.add_created_exploration_id()
    domain_object.add_edited_exploration_id()
    save_domain_object_to_datastore()
```

but I don't see that here.